pypose/basics/ops.py

```diff
@@ -33,7 +33,7 @@ def bmv(input, vec, *, out=None):
     return torch.matmul(input, vec.unsqueeze(-1), out=out).squeeze(-1)
 
 
-class MatrixSquareRoot:
+class msqrt:
```

Comment:
class definition should still use `Msqrt`. Define a new function called `msqrt` which internally calls `Msqrt.apply`, then the users can just call `msqrt(x)` like functions.

Reply:
Because there are many methods of matrix root mean square approximation, whether it is necessary to distinguish different methods by naming. Or use $msqrt$ uniformly?